cmd/server/handler/warehouses_test.go

```diff
@@ -0,0 +1,299 @@
+package controllers_test
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/gin-gonic/gin"
+	"github.com/marcoglnd/mercado-fresco-packmain/cmd/server/routes"
+	"github.com/marcoglnd/mercado-fresco-packmain/internal/warehouses"
+	"github.com/stretchr/testify/assert"
+)
+
+func getPathUrl(url string) string {
+	PATH := "/api/v1"
+	return fmt.Sprintf("%s%s", PATH, url)
+}
+
+func createServer() *gin.Engine {
```

Comment:
What do you think about changing `createServer` to `createServerForWarehouse`, bc all tests need its own server and this way would avoid conflicts between them
```suggestion
func createServerForWarehouse() *gin.Engine {
```